designs/administrator/olm/list-of-operators/list-of-operators.md

```diff
@@ -0,0 +1,45 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# List of Operators
+
+## All projects view
+This is the “all-projects” view which has been modified to now expose more information about the namespaces and without listing out every CSV. The goal of this view is to consolidate the list of operators. There are a few additions to the flow:
+
+1. We have introduced the Operator Namespace column. The Operator Namespace column has 2 descriptors that can live in each row; the first is a single project like “my-project” that is associated with an operator like 3scale. If a namespace appears across All Namespaces it will appear as All Namespaces. In the example of an operator like Jaeger, the text underneath has the namespace name and All Namespaces underneath.
+
+2. In the Managed Namespace column there are three different ways the Namespace information can appear. “My-project” is representative of a single namespace. “All namespaces” is a text only link that is associated with the namespaces that fall across all Namespaces. Thirdly, there is a multi-project view. For example, the Portworx operator falls across multiple projects but not all projects, so in this example the Managed Namespace column will have a popover on click. The namespace names should be listed underneath as seen in the mockup.
+
+![all-projects-view](img/all-projects-view.jpg)
```

Comment:
Since this latest iteration includes a dedicated column for "Managed Namespace", thinking we could remove that "All Namespaces" label underneath the "Operator Namespace".

Reply:
@tlwu2013 @itsptk I've updated these wireframes if you want to give it a final sign off.